### PR TITLE
walk HTML for dependencies and inject into webpack

### PR DESCRIPTION
**Pull request by kylemac**
- intelligently rewrites deps from a dumpdir
- uses cheerio to traverse HTML

feel like there's optimizations to how we're doing the rewrites for files in dumpdirs. open to suggestions, just wanted to get a proof of concept in place first.

### CODE REVIEW

File: lib/plugins/plugin_utils.js [hunk 1]
```diff
@@ -43,6 +43,16 @@ export default {
       }
     }
     done()
+  },
+  resolvePath (compiler, file, opts) {
```
Comment (jenius) [inline lib/plugins/plugin_utils.js]: This is a pretty generic name, would help to have a more specific name or some docs describing what it is doing exactly

**Comment by jenius** [comment]
This is great, certainly we can optimize it but as a proof of concept I'm on board!

**Comment by kylemac** [comment]
@jenius feedback implemented

**Comment by jenius** [comment]
:+1: :+1: :+1:

**Pull request merged**